src/tags/render.js

```diff
@@ -65,7 +65,7 @@ module.exports = function (fractal) {
                     template = file;
                 }
                 else {
-                    template = this.importFile(file);
+                    template = this.template.importFile(file);
```

Comment:
can you give some context for this change as well, what issues does it resolve?

Reply:
same as #35, i added it because i needed it to work while testing things out before the next release, as i'm referencing my version currently.